SMDS_3/include/CGAL/Mesh_complex_3_in_triangulation_3.h

```diff
@@ -328,12 +328,18 @@ class Mesh_complex_3_in_triangulation_3
   const Triangulation& triangulation() const { return tr_; }
 /// @}
 
+#ifndef DOXYGEN_RUNNING
 /// \name Non const access
 /// @{
     /// returns a reference to the triangulation
+    /// \cgalAdvancedBegin
+    /// This function should only be used by advanced users: it merely swaps the triangulation without
+    /// rebuilding critical C3T3 information such as the number of simplexes in complex.
+    /// On the other hand, this is performed by `set_triangulation()`
+    /// \cgalAdvancedEnd
   Triangulation& triangulation() { return tr_; }
```

Comment:
I do not understand:
> it merely swaps the triangulation without...

The function just returns the reference to the non-const triangulation. It does not swap anything. What was the intent?

Reply:
I went too fast, it was meant in the context of using the reference to assign a different triangulation, i.e. something like, "If you use this reference to call the change the triangulation, [...]"